src/libraries/System.Net.Http/src/System/Net/Http/HttpUtilities.cs

```diff
@@ -26,7 +26,11 @@ internal static bool IsHttpUri(Uri uri)
             IsSupportedSecureScheme(scheme);
 
         internal static bool IsSupportedNonSecureScheme(string scheme) =>
```

Comment:
Another case where I'd prefer if we moved this to two partial files, with each doing the desired checks, rather than proliferating use of TARGETS_BROWSER everywhere.  We've avoided that for platforms in general.

Reply:
@kjpou1 In following what we've done before, you'd create `HttpUtilities.Browser.cs` and `HttpUtilities.AnyOS.cs`.  You'll also need to update the csproj accordingly.  There are numerous examples to draw from.

Reply:
Moved this to separate partial files.